docs/building-the-image.md

```diff
@@ -11,22 +11,24 @@ Before you can build the NGINX Kubernetes Gateway, make sure you have the follow
 
 1. Clone the repo and change into the `nginx-kubernetes-gateway` directory:
 
-   ```
+   ```shell
    git clone https://github.com/nginxinc/nginx-kubernetes-gateway.git
+   ```
```

Comment:
I think these can be in a single block? You can copy/paste multiple commands at once